newsletter/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Standard Metadata -->
+    <title>Embark Studios Open Source Newsletter | embark.dev</title>
+    <meta name="description" content="Discover open source repositories in Rust, Go, and more from Embark Studios">
+    <meta name="keywords" content="open source, sponsorship, rust, newsletter">
+    <link rel="icon" href="../favicon.ico">
+
+    <!-- Open Graph Metadata -->
+    <meta property="og:title" content="Embark Studios Open Source Newsletter" />
+    <meta property="og:description" content="Discover open source repositories in Rust, Go, and more from Embark Studios" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://embark.dev" />
+    <meta property="og:image" content="https://embark.dev/img/code.jpg" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@EmbarkStudios" />
+    <meta name="twitter:image" content="https://embark.dev/img/code.jpg">
+
+    <!-- import CSS -->
+    <link rel="stylesheet" href="../style.css">
+
+    <!-- import JS -->
+    <script src="https://kit.fontawesome.com/e541009933.js" crossorigin="anonymous"></script>
+    <script src="https://unpkg.com/vue@2.6.10/dist/vue.min.js"></script>
+    <script src="../script.js"></script>
+  </head>
+<body>
+  <div id="app">
+    <header class="header">
+      <a href="https://embark-studios.com"><img id="logo" src="../img/logo.png"/></a>
+      <div>
+        <a href="https://embark-studios.com" class="fas fa-globe-americas"></a>
+        <a href="https://twitter.com/EmbarkStudios" class="fa fa-twitter"></a>
+        <a href="https://github.com/EmbarkStudios" class="fa fa-github"></a>
+        <a href="https://discord.gg/8TW9nfF" class="fab fa-discord"></a>
+      </div>
+    </header>
+
+  <div class="container">
+    <h1>
+      🗞 Newsletter
+    </h1>
+    <h1>🗞 Newsletter</h1>
+    <p>Stay up to date with new open source projects and developer events from Embark with our developer newsletter.</p>
+    <a class="button-primary" href="https://mailchi.mp/3608b3a1adca/embarkdev">Sign up</a>
+    <h2>Previous Editions:</h2>
+    </ul>
+  </div>
+
+  <section class="category">
+    <div class="newsletter-container container">
+      <a v-bind:href="newsletterUrl(n)" class="project" v-for="n in newsletter">
+        <div class="newsletter-card">
+          <h3 class="title">
+            {{ n.name }}
+          </h3>
+          <p v-html="n.date"></p>
+          <p v-html="n.description"></p>
```

Comment:
I think you can just use `<p>{{ n.date }}</p>` formatting for these 2 lines. The html option is for when the content may include html elements, like links, but these descriptions and dates will always be strings.

Reply:
It's a stylistic choice but I prefer to always use `v-whatever` over `{{ whatever }}` because you don't get a flash of {{}} in the page as the JavaScript is initialising.

I think there might be a `v-bind` or something though? A directive that isn't for injecting HTML. Not used Vue in a while